workbench/src/renderer/components/ResourcesLinks/index.jsx

```diff
@@ -64,17 +73,16 @@ export default function ResourcesTab(props) {
     forumURL = `${FORUM_ROOT}/tag/${tagName}`;
   }
 
-  const userGuideURL = `${UG_ROOT}/${docs}#data-needs`;
-
   const { t, i18n } = useTranslation();
+  const userGuideURL = `${window.Workbench.USERGUIDE_PATH}/${i18n.language}/${docs}`;

```

Comment:
@emlys do you think it's okay to refer to `i18n.language` here instead of passing down some `investSettings` prop to this component? It seems to work fine.

Reply:
I think that should be fine